site/components/colour-box/style.css

```diff
@@ -0,0 +1,62 @@
+@value badgerRed, badgerBlack, badgerWhite, cainYellow, daveGreen, stuBlue from "../../css/_colors.css";
+@value largeScreen, mediumScreen from "../../css/_sizes.css";
+
+.subHeading {
+  composes: serif from "../../css/typography/_fonts.css";
+  color: badgerBlack;
+  display: inline-block;
+  font-size: 24px;
+  line-height: 1.2em;
+  margin: 20px 0 4px;
+  padding: 1px 15px;
+}
+
+.blueSubHeading {
+  composes: subHeading;
+  background-color: stuBlue;
+}
+
+.greenSubHeading {
+  composes: subHeading;
+  background-color: daveGreen;
+}
+
+.yellowSubHeading {
+  composes: subHeading;
+  background-color: cainYellow;
+}
+
+.redSubHeading {
+  composes: subHeading;
+  background-color:badgerRed;
+  color: badgerWhite;
+}
+
+.description {
+  composes: lightSerif from "../../css/typography/_fonts.css";
+  color: #F8F8F8;
+  font-size: 20px;
+  line-height: 1.2em;
+}
+
+a {
+  position: relative;
+}
+
+a .description:hover {
+  border-bottom: 2px solid badgerRed;
+}
```

Comment:
Why does this need an `a` in front of it?

Reply:
@dpiatek At this stage, only one of these colour boxes is a link, with the rest planned to become links over time.  We only want the hover behaviour on the links.

Reply:
that's fine but then you don't need the `a` in that case right? The hover happens on whatever the `.description` is?

Reply:
no, the hover should only happen if the description is within the link.  Three elements have the class 'description' but should not have the hover behaviour as the element is not within a link

Reply:
In that case it should have it's own class ... by doing element selectors in css modules, like the one above, we essentially defeat the purpose of CSS modules

Reply:
ok cool, updated!